RAMP/common.py

```diff
@@ -1,7 +1,14 @@
 
 from __future__ import print_function
 import sys
+import os
 
+try:
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), "../../readies"))
+    sys.path.insert(0, "/w/rafi_1/readies")
+    import paella
```

Comment:
Why paella is needed here?

Reply:
It's for debugging, so that you can put `BB()` in the code and run with `BB=1 python ...` and stop on breakpoints. `BB()` will only have an affect if env var `BB` is present.